modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -85,5 +86,45 @@ def report msg, response, error_only
       logger.error "Dnscmd failed:\n" + (response.is_a?(Array) ? response.join("\n") : "Response was not an array! #{response}")
       raise Proxy::Dns::Error.new("Unknown error while processing '#{msg}'")
     end
+
+    def match_zone(fqdn)
+      dns_zones = ::Proxy::Dns::Plugin.settings.dns_zones || enum_dns_zones
```

Comment:
I don't think this is a good idea -- why have such an override at all? Esp. considering potential issues -- the list getting out of sync, wrong entries in the list, etc. 

I'm ok with retrieving of dns zones on every call for now however.